### PR TITLE
Advising Dashboard Frontend Enhancements #1

### PR DESCRIPTION
**Pull request by jwfung**

### CODE REVIEW

File: static/css/timetable/framework/page_layout.scss [hunk 1]
```diff
@@ -56,25 +56,25 @@ textarea {
   }
 }
 
-.comments-forum {
+.comment-forum {
   background-color: $gfff;
   bottom: auto;
   height: 100%;
-  left: calc(100% - 3000px);
+  left: calc(100% - 700px);
```
Comment (miaboloix) [inline static/css/timetable/framework/page_layout.scss:63]: i feel like 500px might look better? the chat seems a bit large?

**Comment by miaboloix** [comment]
In the future, lets make smaller PRs if possible. We could have had a few different PRs from these changes - one PR for the top bar change, one PR for the comments-forum fix, etc.

File: static/css/timetable/framework/page_layout.scss (hunk 1 above)
Comment (miaboloix) [reply static/css/timetable/framework/page_layout.scss:63]: i fix this in my pr

**Review by miaboloix — APPROVED** [review]
I'm approving - lets have the next PRs be smaller and work on one improvement at a time. This will also help you all work individually on your own PRs. Some example PRs we could work on: (1) Create message components (2) Fix Comment-Forum-Container UI (ex: add border, model spacing off of existing components, make outline of chat)

**Review by Boccca2014 — APPROVED** [review]

**Pull request merged**